srcCpp/Infrastructure_pro.cxx

```diff
@@ -5,6 +5,19 @@
 
 #include "Infrastructure_pro.h"
 
+// Since std::to_string is not defined in c++98
+// we will define it here for VxWorks
+#ifdef RTI_VXWORKS
```

Comment:
I think this might be use as well for Micro, i would move it to infrastructure_common, what do you think?

Reply:
Sure, but for what I have seen so far,  `infrastructure_common.h` includes `infrastructure_[micro | pro].h` How will Micro access this then?

Reply:
I have tried to move it to `Infrastructure_common.cxx` and it has failed compiling